plugins/google_drive/komand_google_drive/actions/create_file_in_folder/action.py

```diff
@@ -0,0 +1,39 @@
+import komand
+from .schema import CreateFileInFolderInput, CreateFileInFolderOutput, Input, Output, Component
+
+# Custom imports below
+from googleapiclient.http import MediaIoBaseUpload
+from mimetypes import guess_type
+from io import BytesIO
+from base64 import b64decode
+from googleapiclient import errors
+
+
+class CreateFileInFolder(komand.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="create_file_in_folder",
+            description=Component.DESCRIPTION,
+            input=CreateFileInFolderInput(),
+            output=CreateFileInFolderOutput(),
+        )
+
+    def run(self, params={}):
+        filename = params.get("file").get("filename")
+        file_bytes = params.get("file").get("content")
+        file_metadata = {"name": filename, "parents": [params.get(Input.FOLDER_ID)]}
+
+        mime_type = guess_type(filename)
+        if mime_type:
+            mime_type = guess_type(filename)[0]
+            if not mime_type:
```

Comment:
The indentation for the if block might be in the wrong place. Also since there's only two cases here, should the last if be an else statement?

Reply:
I corrected the indentation for this if block but did not change the second if statement to else because `guess_type(filename)[0]` may return `None` if the user gives an empty string as filename or no file extension, or if the `guess_type` method is not able to determine the file mime type based on its extension.